src/common/Axis/RadialAxis/RadialAxisTickSeries/RadialAxisTickLabel.tsx

```diff
@@ -58,16 +58,6 @@ export interface RadialAxisTickLabelProps {
    * Format of the label.
    */
   format?: (value: any, index: number) => any | string;
-
-  /**
-   * Start angle for the first value.
-   */
-  startAngle?: number;
```

Comment:
We don't need these anymore?

Reply:
just for the labels. i don't think it makes sense.. it's rotating things like this 
![image](https://github.com/reaviz/reaviz/assets/1513140/d72cff74-ef19-4628-a1f3-45c528bca2c0)

Reply:
Agree with @steppy452. Since the ticks(containing the lines and labels) are rotated themselves so probably we may not want to rotate the labels.